circus-train-s3-s3-copier/src/test/java/com/hotels/bdp/circustrain/s3s3copier/S3S3CopierOptionsTest.java

```diff
@@ -96,4 +96,24 @@ public void getCannedAclDefaultIsNull() throws Exception {
     assertNull(options.getCannedAcl());
   }
 
+  @Test
+  public void getMaxCopyAttempts() throws Exception {
+    copierOptions.put(S3S3CopierOptions.Keys.MAX_COPY_ATTEMPTS.keyName(), 3);
+    S3S3CopierOptions options = new S3S3CopierOptions(copierOptions);
+    assertThat(options.getMaxCopyAttempts(), is(3));
+  }
+
+  @Test
+  public void getMaxCopyAttemptsDefaultIsOne() throws Exception {
+    S3S3CopierOptions options = new S3S3CopierOptions(copierOptions);
+    assertThat(options.getMaxCopyAttempts(), is(3));
+  }
+
+  @Test
+  public void getMaxCopyAttemptsDefaultIsOneIfLessThanOne() throws Exception {
```

Comment:
This one needs renaming too.

Reply:
Shouldn't there be a validation check at the config level that the numAttempts can't be less than one ?